test/integration/targets/ecs_certificate/tasks/main.yml

```diff
@@ -198,6 +199,17 @@
         - example4_result.tracking_id > 0
         - example4_result.serial_number is string
 
+  # For bug 61738, verify that the full chain is valid
+  - name: Verify that the full chain path can be successfully imported
+    openssl_pkcs12:
+      action: export
+      certificate_path: '{{ example4_cert_path }}'
+      other_certificates: '{{ example4_full_chain_path }}'
```

Comment:
Please note that `other_certificates` only reads the first certificate of every filename put in that list. So if your chain starts with the certificate itself, this is simply the certificate.

Also note that the module simply puts the certificates you give it into a PKCS12 file. Successfully creating the file does not imply that the chain is valid (it essentially only checks whether the files contain a syntactically valid certificate).

To check whether a chain is valid, you have to use the `openssl` binary like here: https://github.com/felixfontein/acme-certificate/blob/master/tasks/main.yml#L194